examples/vmi-macvtap.yaml

```diff
@@ -33,8 +33,11 @@ spec:
     name: containerdisk
   - cloudInitNoCloud:
       userData: |-
-        #!/bin/bash
-        echo "fedora" |passwd fedora --stdin
-        yum install -y nginx
-        systemctl enable --now nginx
+        #cloud-config
+        password: fedora
+        chpasswd: { expire: False }
```

Comment:
When using the `fedora-with-test-tooling-container-disk` image, I think we no longer need this part.
I suggest to add a commit that removes it.

In some of the scenarios below, this will make the `userData` section redundant.

Reply:
I think we should use vanilla images, and provide examples on how to get something useful out of those, rather than use a customized image with "black magic" in it, that does not require customization.